daemon/plugin/vendorplugin.go

```diff
@@ -94,5 +102,17 @@ func (g *GrpcPlugin) ensureConnected() error {
 	g.conn = conn
 
 	g.client = pb.NewLifeCycleServiceClient(conn)
+	g.opiClient = opi.NewBridgePortServiceClient(conn)
 	return nil
 }
+
+func (g *GrpcPlugin) CreateBridgePort(createRequest *opi.CreateBridgePortRequest) error {
+	g.ensureConnected()
+	_, err := g.opiClient.CreateBridgePort(context.TODO(), createRequest)
```

Comment:
this is where we will forward req to vsp right?

Reply:
yes